northstar.toml

```diff
@@ -20,3 +16,15 @@ loop_control = "/dev/loop-control"
 loop_dev = "/dev/loop"
 device_mapper = "/dev/mapper/control"
 device_mapper_dev = "/dev/dm-"
+
+[debug.runtime]
+log_level = "DEBUG"
+disable_mount_namespace = false
+
+# [debug.strace]
+# log_to_file = true
+# flags = "-v -f -s 256"
```

Comment:
Maybe we should not just comment out a section to disable it.
In case of `debug.runtime` it is also misleading since commenting it out will not turn off logging but select a default level.
e.g. for `[debug.strace]` it could be:
```
enabled = false
...
```

Reply:
For the logs I agree that we can make the default explicit because "something" is done from the default.

For `strace` I'd like to keep it optional since if it's not there - *nothing* is done.

Reply:
Move `log_level` to the main section.